.github/workflows/php.yml

```diff
@@ -0,0 +1,97 @@
+name: PHP
+
+on: [push, pull_request]
+jobs:
+    php-cs-fixer:
+        name: PHP CS Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.1'
+
+            - name: Get Composer Cache Directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache Composer Directory
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - name: Validate composer config
+              run: composer validate --strict
+
+            - name: Composer Install
+              run: composer install
+
+            - name: Run PHPCSFixer
+              run: ./vendor/bin/php-cs-fixer fix --dry-run --diff
+
+    phpstan:
+        name: PHP Static Analysis
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.1'
+
+            - name: Get Composer Cache Directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache Composer Directory
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - name: Composer Install
+              run: composer install
+
+            - name: Run phpstan
+              run: ./vendor/bin/phpstan analyse
```

Comment:
What about testing for multiple php versions? https://phpstan.org/config-reference#miscellaneous-parameters